Routing/SegmentVariant/Literal.cs

```diff
@@ -0,0 +1,19 @@
+﻿#pragma warning disable 660,661
+
+namespace Routing.SegmentVariant
+{
+    [Equals]
+    internal class Literal : ISegmentVariant
+    {
+        internal Literal(string identifier)
+        {
+            Identifier = identifier;
+        }
+
+        internal string Identifier { get; }
+
+        public static bool operator==(Literal left, Literal right) => Operator.Weave(left, right);
```

Comment:
Missing spaces around the operator

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/f877a13ceca04a503a59256b3e286f6a639b18c0

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/cb332a60a6bb41fd2d749a6ea5025562a38fcb7d